src/linearmixedmodel.jl

```diff
@@ -466,7 +466,7 @@ function Base.getproperty(m::LinearMixedModel, s::Symbol)
     elseif s == :b
         ranef(m)
     elseif s == :feterms
-        filter(Base.Fix2(isa, FeMat), getfield(m, :allterms))
+        convert(Vector{FeMat{T}}, filter(Base.Fix2(isa, FeMat), getfield(m, :allterms)))
```

Comment:
This also returns the response because the response is stored as an `FeMat`. Is that the desired behavior (which is no change from previous behavior of course)?

Reply:
Ah, yes there's also the non-exported `fetrm`.